src/pilot/assign_role.py

```diff
@@ -1288,14 +1276,21 @@ def ensure_node_is_powered_off(drac_client):
 def change_physical_disk_state_wait(
         node_uuid, ironic_client, drac_client, mode,
         controllers_to_physical_disk_ids=None):
-    reboot_required, job_ids = change_physical_disk_state(
-        drac_client, mode, controllers_to_physical_disk_ids)
+
+    physical_disk_status = drac_client.change_physical_disk_state(
+        mode, controllers_to_physical_disk_ids)
+
+    job_ids = []
+    if physical_disk_status['commit_required_ids']:
+        for controller in physical_disk_status['commit_required_ids']:
```

Comment:
Would you mind changing "controller" to "controller_id" above?  There is a controller object, but the variable above actually contains just an id and not the object.

Reply:
sure i will do the change